src/components/wallet/NewMnemonicConfirm.tsx

```diff
@@ -65,13 +67,8 @@ const MnemonicConfirmScreen: React.FC<{ mnemonic: string; onConfirm: Function }>
 
   init()
 
-  const isSelected = useCallback(
-    (id: string) => {
-      const res: WordType | undefined = wordsList.find((e: WordType) => e._id === id)
-      return !!res?.selected
-    },
-    [wordsList]
-  )
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why do we need this ^ here?

Reply:
react's preset linter can not infer returned type from factory function return type and throws a warning that deps list is unknown